example/widgetbook/pages/components/progress_widgetbook.dart

```diff
@@ -33,5 +33,6 @@ Widget progressCircleUseCase(BuildContext context) => WidgetbookScaffold(
           labelBuilder: enumLabelBuilder,
         ),
         onCancel: context.knobs.boolean(label: "Can Cancel") ? () {} : null,
+        label: context.knobs.stringOrNull(label: 'Label'),
```

Comment:
can you also add maxValue into widgetbook please

Reply:
Yes sirrr